pkg/registry/serving/composer.go

```diff
@@ -337,14 +350,18 @@ func composeInitContainer(sdep *seldonv1.SeldonDeployment) error {
 		}
 
 		container := &corev1.Container{
-			Name:  "model-initializer",
+			// mimics the behavior of seldon model initializer for it will disable the default init container injection
+			// in case of a pre-packaged server implementation was selected: https://github.com/SeldonIO/seldon-core/blob/0bd83773228a18e7f376270f4b85cbef69395b8f/operator/controllers/model_initializer_injector.go#L142
+			// the default name will be generated here:
+			// https://github.com/SeldonIO/seldon-core/blob/0ef45fd234a674fc9b6c8d034cd2e42b4c9ebd05/operator/controllers/model_initializer_injector.go#L118
+			Name:  p.Name + "-" + "model-initializer",
 			Image: viper.GetString(envModelInitializerImage),
 			Args:  []string{p.Graph.ModelURI, modelMountPath},
 			// Get username and password from environment
 			// Here AWS_SECRET_ACCESS_KEY and AWS_ACCESS_KEY_ID are used
 			// because Seldon Core does not support renaming the environment variable name.
 			// it is used in ormb-storage-initializer.
-			// please refenence https://github.com/kleveross/ormb/blob/master/cmd/ormb-storage-initializer/cmd/pull-and-export.go#L47
+			// please refenence https://github.com/kleveross/ormb/blob/4ed110bdcd63540b7feaa179ebed2cb94c2aaba2/cmd/ormb-storage-initializer/cmd/pull-and-export.go#L47
```

Comment:
Why we need replace `master` with `4ed110bdcd63540b7feaa179ebed2cb94c2aaba2`

Reply:
Master will be updated frequently, then we can use a commit to keep the link always pointing to the right code block.

Reply:
Got it, SGTM